### ISSUE TITLE
Deploy: "App not found" while app name is correct

### ISSUE THREAD

**Issue by obrunsmann** [Baqend/js-sdk#15]
When I try to deploy via CLI I get the message

> App not found.

while app name is fully correct. I used the CLI like: `baqend deploy -F build app-name`. `baqend apps` lists my app correctly and the possibility of typing errors is excluded.

**Comment by fbuecklers** [comment]
Hi,

I guess I have found the issue which causes your error. I think you want to specify the folder which should be deployed. You must specify the folder with a lower case `-f` flag...

The upper case `-F` just says you only want to deploy files and doesn't accept a parameter. That's why the CLI guess that `build` is your app name where you want to deploy to.
As described in the CLI command docs:
```
> baqend deploy --help
...
    -F, --files                deploy files
    -f, --file-dir <dir>       path to file directory [default:www] (default: www)
```

I have added the app name to the `App not found.` error message, to make the error more clear in the future.
